hatchet_sdk/context/context.py

```diff
@@ -179,11 +189,11 @@ def __init__(
         # Check the type of action.action_payload before attempting to load it as JSON
         if isinstance(action.action_payload, (str, bytes, bytearray)):
             try:
-                self.data = json.loads(action.action_payload)
+                self.data = cast(dict[str, Any], json.loads(action.action_payload))
```

Comment:
does the payload need to be a dictionary? are there cases where it could be other valid json, like a list or a string or something?

Reply:
No, at the moment the action payload is always a JSON object at the top level.